setup/delivery_carrier_partner/odoo/addons/delivery_carrier_partner

```diff
@@ -0,0 +1 @@
+../../../../delivery_carrier_partner
```

Comment:
This should go in a previous commit, as it's automatic.

Reply:
I delete this part?

Reply:
OK, it's not too important. Let's keep it this way.